tools/bazel.bzl

```diff
@@ -14,13 +14,29 @@ def _py_library(name, **kwds):
 def _requirement(_pypi_project):
     return None
 
+def pytorch_cc_test(name, srcs, tags = [], **kwargs):
+    """PyTorch cc test rule.
+
+    One of the reason to proxy all tests through this central location
+    is ability to change the behavior on all of them instead of one-by-one.
+
+    For example, sombody using Remote Build Execution can add exec_properties for tests here.
+    """
+    if "gpu-required" in tags:
```

Comment:
This is not yet supported by bazel, see https://github.com/bazelbuild/bazel/issues/10799

Reply:
Sorry, I think I misunderstood.
This would not achieve the objective to

> Allows RBE users to have a central place for customizations of cc tests.

There are properties like Platform https://github.com/bazelbuild/remote-apis/blob/main/build/bazel/remote/execution/v2/remote_execution.proto#L520 that are implementation specific.
Without indirection, we cannot use a single place to customize such properties.

For example at Cruise, we have a property for "worker pool" which determines how many GPUs we need on a worker.